src/server/rdb_load.cc

```diff
@@ -2508,9 +2507,8 @@ void RdbLoader::PerformPostLoad(Service* service) {
     return;
 
   // Rebuild all search indices as only their definitions are extracted from the snapshot
-  Transaction::RunOnceAsCommand(cmd, [](auto* trans, auto* es) {
-    es->search_indices()->RebuildAllIndices(trans->GetOpArgs(es));
-    return OpStatus::OK;
+  shard_set->AwaitRunningOnShardQueue([](EngineShard* es) {
+    es->search_indices()->RebuildAllIndices(OpArgs{es, nullptr, DbContext{0, GetCurrentTimeMs()}});
```

Comment:
Maybe I should pass the shard + db context separately to avoid using OpArgs with a null transaction field, but it's really a nit